docs/web/experimental.css

```diff
@@ -69,42 +69,22 @@ color: black;
 }
 
 div.flex {
-display:flex; justify-content:flex-start; align-items: flex-start;; flex-direction:row; flex-wrap: wrap;
+/*  display:flex; justify-content:flex-start; align-items: flex-start;; flex-direction:row; flex-wrap: wrap;*/
+display: inline-grid;
+grid-template-columns: auto auto;
```

Comment:
@arrowarchive take a look at this: [A Very Handy, Illustrative, and Complete Guide to CSS Grid](https://css-tricks.com/snippets/css/complete-guide-grid/)